src/Ticket.php

```diff
@@ -3914,12 +3921,12 @@ public static function getDefaultValues($entity = 0)
 
        // Set default values...
         return  ['_users_id_requester'       => $users_id_requester,
-            '_users_id_requester_notif' => ['use_notification'  => [$default_use_notif],
+            '_users_id_requester_notif' => ['use_notification'  => [(string) ($default_use_notif & $user_notification_enable)],
                 'alternative_email' => ['']
             ],
             '_groups_id_requester'      => 0,
             '_users_id_assign'          =>  $users_id_assign,
-            '_users_id_assign_notif'    => ['use_notification'  => [$default_use_notif],
+            '_users_id_assign_notif'    => ['use_notification'  => [(string) ($default_use_notif & $user_notification_enable)],
```

Comment:
```$default_use_notif``` contain an ```int```
```$requester_notification_enable``` contain a ```bool```

```(string) (1 && true)  ==> string "1"```
```(string) (1 && false)  ==> string ""``` we want 0 here (break TU) otherwise

```(string) (1 & true)  ==> string "1"```
```(string) (1 & false)  ==> string "0"```